src/shared/modules/app/appDuck.ts

```diff
@@ -46,6 +47,10 @@ export const inWebEnv = (state: any) => getEnv(state) === WEB
 export const inCloudEnv = (state: any) => getEnv(state) === CLOUD
 export const inWebBrowser = (state: any) => [WEB, CLOUD].includes(getEnv(state))
 export const inDesktop = (state: any) => getEnv(state) === DESKTOP
+export const getGitRevision = (state: any): string | null =>
```

Comment:
Add GlobalState type?

Reply:
App wasn't added properly on the global state, but I went ahead and added it now :+1: